cmus.svg

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Ideally this should be part of upstream repo too.

Reply:
Yes, as I said there is an issue upstream about it (the icon is from that issue's reporter).